src/shared/errors/index.js

```diff
@@ -189,6 +191,16 @@ export class ActionFunctionOptionError extends ActionOptionErrorBase {
     }
 }
 
+export class ActionInvalidObjectPropertyError extends TestRunErrorBase {
+    constructor (objectName, propertyName, availableProperties) {
+        super(TEST_RUN_ERRORS.actionInvalidObjectPropertyError);
+
+        this.objectName = objectName;
+        this.propertyName  = propertyName;
+        this.availableProperties  = arraySort.call(availableProperties);
```

Comment:
To simplify the code, I propose to order available properties in the `Assignable._getReportedProperties`.